tests/components/zeroconf/test_init.py

```diff
@@ -114,6 +114,43 @@ async def test_setup_without_default_interface(hass, mock_zeroconf):
     assert mock_zeroconf.called_with()
 
 
+async def test_setup_without_ipv6(hass, mock_zeroconf):
+    """Test without ipv6."""
+    with patch.object(hass.config_entries.flow, "async_init"), patch.object(
+        zeroconf, "HaServiceBrowser", side_effect=service_update_mock
+    ):
+        mock_zeroconf.get_service_info.side_effect = get_service_info_mock
+        assert await async_setup_component(
+            hass, zeroconf.DOMAIN, {zeroconf.DOMAIN: {CONF_IPV6: False}}
+        )
+
+    assert mock_zeroconf.called_with(ip_version=IPVersion.V4Only)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)